sdks/java/io/google-cloud-platform/README.md

```diff
@@ -0,0 +1,70 @@
+# Google Cloud Platform IOs
```

Comment:
I agree that we will eventually want a README.md, but I'd prefer to defer writing it until we move the rest of the GCP-related classes into the `io.gcp` package.